mailet/standard/src/main/java/org/apache/james/transport/matchers/RelayLimit.java

```diff
@@ -21,29 +21,31 @@
 
 package org.apache.james.transport.matchers;
 
-import org.apache.mailet.base.MailetUtil;
-import org.apache.mailet.base.RFC2822Headers;
-import org.apache.mailet.base.GenericMatcher;
-import org.apache.mailet.Mail;
-import org.apache.james.core.MailAddress;
+import java.util.Collection;
+import java.util.Enumeration;
 
 import javax.mail.Header;
 import javax.mail.MessagingException;
 import javax.mail.internet.MimeMessage;
 
-import java.util.Collection;
-import java.util.Enumeration;
+import org.apache.james.core.MailAddress;
+import org.apache.mailet.Mail;
+import org.apache.mailet.base.GenericMatcher;
+import org.apache.mailet.base.MailetUtil;
+import org.apache.mailet.base.RFC2822Headers;
 
 /**
  * Matches mail which has been relayed more than a given number of times.
  * @version 1.0.0, 1/5/2000
  */
 
 public class RelayLimit extends GenericMatcher {
-    int limit = 30;
+    private int limit;
 
     public void init() throws MessagingException {
-        limit = MailetUtil.getInitParameterAsStrictlyPositiveInteger(getCondition());
+        limit = MailetUtil.integerConditionParser()
```

Comment:
what about introducing shortcuts for these very common cases, like : ̀MailetUtil.parseStrictlyPositiveInteger()` ?